fabric-common/lib/BlockDecoder.js

```diff
@@ -1624,9 +1692,9 @@ const HeaderType = class {
 			case 3:
 				result = decodeEndorserTransaction(proto_data);
 				break;
-			case 9:
-				result = decodeTokenTransaction(proto_data);
-				break;
+			// case 9:
+			// 	result = decodeTokenTransaction(proto_data);
+			// 	break;
 			default:
```

Comment:
same here

Reply:
I left it as I heard rumblings of token coming back, but your right I should just remove it